src/lib.rs

```diff
@@ -92,6 +92,27 @@
 //! }
 //! ```
 //!
+//! # Available logging frameworks
+//!
+//! In order to produce log output executables have to utilize
```

Comment:
"Utilize" is not the right word. See https://english.stackexchange.com/questions/143941/when-to-use-use-and-when-to-use-utilize-in-a-sentence/.